mappings/net/minecraft/world/gen/chunk/Blender.mapping

```diff
@@ -53,9 +61,10 @@ CLASS net/minecraft/class_6748 net/minecraft/world/gen/chunk/Blender
 	METHOD method_40027 (IILorg/apache/commons/lang3/mutable/MutableDouble;Lorg/apache/commons/lang3/mutable/MutableDouble;Lorg/apache/commons/lang3/mutable/MutableDouble;Ljava/lang/Long;Lnet/minecraft/class_6749;)V
 		ARG 5 chunkPos
 		ARG 6 data
-	CLASS class_6781
-		METHOD get (Lnet/minecraft/class_6749;III)D
+	CLASS class_6781 BlendingSampler
+		METHOD get sample (Lnet/minecraft/class_6749;III)D
```

Comment:
Please do not rename those.

Reply:
Is that for technical reasons or a convention?

Reply:
Both, I think. They aren't assigned intermediary names, so things might break.